crates/jstzd/tests/jstzd_test.rs

```diff
@@ -0,0 +1,64 @@
+mod utils;
+use octez::r#async::endpoint::Endpoint;
+use octez::r#async::node_config::OctezNodeConfigBuilder;
+use octez::unused_port;
+use utils::retry;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn jstzd_test() {
+    let rpc_endpoint = Endpoint::localhost(unused_port());
+    let mut config_builer = OctezNodeConfigBuilder::new();
+    config_builer.set_rpc_endpoint(&rpc_endpoint);
```

Comment:
Nit: 
```suggestion
    let mut config_builer = OctezNodeConfigBuilder::new().set_rpc_endpoint(&rpc_endpoint);
```

Reply:
`OctezNodeConfigBuilder::new()` needs to be held at a temporary place so that it lives long enough. I've replaced this with `OctezNodeConfigBuilder::new()....build()`.

Reply:
Ah, the builder patter was implemented with `&mut self` instead of `mut self`... I think a consuming builder would have been nicer to work with since octez configurations are relatively simple